.gitignore

```diff
@@ -36,4 +36,5 @@ yarn-debug.log*
 .yarn-integrity
 
 .docker-sync/*
-.docker-sync
\ No newline at end of file
+.docker-sync
+app/models/concerns/slug_history.rb
```

Comment:
hm did you mean to add this in here? did you accidentally run `git rm`

Reply:
oops! sorry. i will remove this change from this pr.